packages/node/src/login/oidc/incomingRedirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -471,7 +471,10 @@ describe("AuthCodeRedirectHandler", () => {
     it("throws if the iss parameter does not match stored issuer", async () => {
       const mockedStorage = mockDefaultRedirectStorage();
       const mockedTokens = mockDpopTokens();
-      setupOidcClientMock(mockedTokens);
+      // oidc-client will throw if the iss parameter mismatches.
+      setupOidcClientMock(mockedTokens, () => {
```

Comment:
It is a bit unfortunate to have to do this, rather than mocking out the responses from the OpenID Provider to `oidc-client`